utils/NamedType.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NAMEDTYPE
+#define NAMEDTYPE
+
+template <typename T, typename Parameter>
+class NamedType
+{
+public:
+    explicit NamedType(T const& value) : value_(value) {}
+    T& get() { return value_; }
+    T const& get() const { return value_; }
+private:
+    T value_;
+};
+
+using Temperature = NamedType<double, struct TemperatureParameter>;
+using Humidity    = NamedType<double, struct HumidityParameter   >;
+
+using DHTData  = NamedType<uint8_t, struct DHTDataParameter >;
+using DHTType  = NamedType<uint8_t, struct DHTTypeParameter >;
+using DHTPower = NamedType<uint8_t, struct DHTPowerParameter>;
+
+using RadioNum   = NamedType<int8_t, struct RadioNumParameter  >;
```

Comment:
If I'm not mistaken previosly radio num was `uint16_t `. Could you please check other types too?